### DISCUSSION TITLE
blokada w comodo dragon

### DISCUSSION THREAD

**Discussion by blocker999** [FiltersHeroes/KAD#1979] [Pytania i odp] [answered]
Na stronie `czasdzieci.pl/warszawa/`
gdy po klikam w książke "gwiazdkowy prosiaczek" blokuje link

`chrome-extension://ffjgpapimgnmibnacmeilgjefnoofefp/warning/alert.html?type=Malware&targeturl=https://bit.ly/3czXNHt`

![cd](https://user-images.githubusercontent.com/23176832/143678666-e5e1e871-ed81-4530-8d2c-fdeed8c87ca0.png)

**Comment by xMaap** [comment]
Comodo nie powinien tego blokować.
`https://bit.ly/3czXNHt`
Gdzie przekierowuje do
`https://www.empik.com/gwiazdkowy-prosiaczek-rowling-j-k,p1275974159,ksiazka-p`
Może Comodo ogólnie blokuje skracacz od bitly.com.

**Comment by krystian3w** [comment, marked as answer]
Zgłoś to do twórcy przeglądarki/dodatku "Online Security Pro", KAD to nie Matka Teresa z Kalkuty do wyręczania biednego Blocker999, bo zna tylko francuski.

ID dodatku wskazuje że masz: https://chrome.google.com/webstore/detail/online-security-pro/ffjgpapimgnmibnacmeilgjefnoofefp

Spróbuj go chamsko odinstalować z Dragon i problem pewnie zniknie.

**Comment by xMaap** [comment]
@blocker999 ja bym uważał z używaniem tej przeglądarki z powodu słabego wsparcia.
Comodo Dragon Internet Browser ostatnio dostał aktualizację we wrześniu co za tym idzie ma pełno luk.

**Reply by krystian3w** [reply]
W Chromium ostatnio luki 0 day są w ilości 1 na miesiąc jak nie więcej.

**Reply by hawkeye116477** [reply]
Czyli przesiadł się z przeglądarki, która ma co raz słabsze wsparcie (Waterfox Classic) do kolejnej, która ma słabe.

#Vivaldilepszy :smile:
